pulsar/schema.go

```diff
@@ -37,27 +38,27 @@ import (
 type SchemaType int
 
 const (
-	NONE        SchemaType = iota //No schema defined
```

Comment:
Please do not submit format changes that are not related to this PR.

Reply:
OK, but I think it is better to keep a same comment format for a project, so I configured my IDE to run gofmt when I save the files, I have to update my IDE config when I prepare a PR, and revert the config when I do my own job, it not so convinient...

Reply:
Hi shibd, I have revert the comment

Reply:
Yes, If there is a good way to automate, you can contribute to the code format alone,